src/replication.c

```diff
@@ -1110,7 +1112,9 @@ send_replica_handshake_query(replica_t *replica, spec_t *spec)
 	TAILQ_INSERT_TAIL(&replica->mgmt_cmd_queue, mgmt_cmd, mgmt_cmd_next);
 	MTX_UNLOCK(&replica->r_mtx);
 	if (write(replica->mgmt_eventfd1, &num, sizeof (num)) != sizeof (num)) {
-		REPLICA_NOTICELOG("Failed to inform to mgmt_eventfd for replica(%p)\n", replica);
+		REPLICA_NOTICELOG("Failed to inform to mgmt_eventfd for "
+		    "replica(%lu) (%s:%d) mgmt_fd:%d\n", replica->zvol_guid,
```

Comment:
Here zvol_guid will be `0` for all replica since ISTGT haven't initialized data connection at this point.